<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Requettes</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>741</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Liste de Requetes</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>290</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer selection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>290</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier Selection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>30</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>172</x>
         <y>30</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Type de requete</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>30</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Filtrer par service</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>290</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Extraire PDF</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>631</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>390</y>
        <width>491</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter Requete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Equipe de maintenance</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>741</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Liste de Requetes</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>711</width>
         <height>171</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Chef Equipe</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Service Specialise</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Effectif</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre requetes effectuees</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>14587</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Sadok Mohamed</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>Reparation Vehicules</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>54875</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>Slimani Mourad</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>Reparation Electrique</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="1" column="4">
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>78545</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>Ammar Sinda</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>Informatique</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item row="2" column="4">
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>46484</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>Ben Khlifa Mariem</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>BioTech</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item row="3" column="4">
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>290</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer selection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>290</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier selection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>361</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Filtrer par service</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>290</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Extraire PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>380</y>
        <width>491</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter Equipe</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>481</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Formulaire Equipe</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_6">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>190</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>340</y>
         <width>311</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Effectif</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Specialite</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nb Requettes</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>230</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>310</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>421</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Chef d'equipe</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>481</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Choix</string>
      </property>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImporter_Document">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Document"/>
    <addaction name="actionExport_Pdf"/>
    <addaction name="actionSave_changes"/>
    <addaction name="actionSave_document"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuImporter_Document"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Document">
   <property name="text">
    <string>Import Document</string>
   </property>
  </action>
  <action name="actionExport_Pdf">
   <property name="text">
    <string>Export Pdf</string>
   </property>
  </action>
  <action name="actionSave_changes">
   <property name="text">
    <string>Save changes</string>
   </property>
  </action>
  <action name="actionSave_document">
   <property name="text">
    <string>Save document</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
